Formula/r/readsb.rb

```diff
@@ -0,0 +1,60 @@
+class Readsb < Formula
+  desc "ADS-B decoder swiss knife"
+  homepage "https://github.com/wiedehopf/readsb"
+  url "https://github.com/wiedehopf/readsb/archive/refs/tags/v3.14.1666.tar.gz"
+  sha256 "395063fdc4ba70af14b5a782c230b94dd2bc7c19a6922c935ea8465ddaddc717"
+  license "GPL-3.0-or-later"
+  revision 1
+
+  depends_on "pkgconf" => :build
+  depends_on "librtlsdr"
+  depends_on "ncurses"
+  depends_on "zstd"
+
+  on_macos do
+    depends_on "libusb"
+  end
+
+  def install
+    with_env(RTLSDR: "yes") { system "make" }
+    bin.install "readsb"
```

Comment:
Is there no `make install`?
```suggestion
    ENV["RTLSDR"] = "yes"
    system "make"
    bin.install "readsb"
```

Reply:
No. The makefile does not have `install` and the manual build [instructions](https://github.com/wiedehopf/readsb?tab=readme-ov-file#building-manually) don't ask for it.